redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  labels:
+    app: redirector-k8s-io
+    version: dirty
+spec:
+  revisionHistoryLimit: 2
+  replicas: 2
+  strategy:
+    rollingUpdate:
```

Comment:
This block doesn't seem to make sense to me

Reply:
Happy to go with the default of 25% maxUnavailable and 25% maxSurge aka we remove this block.